suite-native/atoms/src/TouchableSwitchRow.tsx

```diff
@@ -0,0 +1,66 @@
+import { ReactNode } from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { Icon, IconName } from '@suite-native/icons';
+
+import { Box } from './Box';
+import { Card } from './Card/Card';
+import { HStack, VStack } from './Stack';
+import { Switch } from './Switch';
+import { Text } from './Text';
+
+type TouchableSwitchRowProps = {
+    isChecked: boolean;
+    onChange: (value: boolean) => void;
+    text: ReactNode;
+    accessibilityLabel: string;
+    description?: ReactNode;
+    iconName: IconName;
```

Comment:
Some components use `iconName`, some `icon`. I'd go with `icon` and probably make it the first property in the type.